lib/ReactViews/Mobile/MobileMenuItem.jsx

```diff
@@ -34,7 +37,9 @@ export default function MobileMenuItem(props) {
 MobileMenuItem.propTypes = {
   href: PropTypes.string,
   onClick: PropTypes.func,
-  caption: PropTypes.string
+  caption: PropTypes.string,
+  showMobileIcon: PropTypes.bool,
+  icon: PropTypes.string
```

Comment:
I think this should be `PropTypes.object` as you are passing in an object here

https://github.com/TerriaJS/terriajs/blob/d1515b43b0c24350717699c0ac056a3d0d6f9484/lib/ReactViews/Map/Panels/LangPanel/LangPanel.tsx#L41

Reply:
Which gets passed in here
https://github.com/TerriaJS/terriajs/blob/d1515b43b0c24350717699c0ac056a3d0d6f9484/lib/ReactViews/Map/Panels/MobilePanel.jsx#L27-L31

Reply:
sorry i don't understand, are you suggesting that MobileMenuItem should have a single prop with the caption and the icon as keys?

Reply:
Sorry I meant `icon` should be `PropTypes.object` - as you are passing in an `object` through:
- `MenuPanel` `mobileIcon` prop -> `MobileMenuItem` `icon` prop

I was getting an error in console about the prop being an `object`, even though it's prop type is `string`